assets/i18n/active.en.toml

```diff
@@ -0,0 +1,85 @@
+# General usage
```

Comment:
We gebruiken nu TOML, maar zie dat YAML ook ondersteund wordt. Mss wel iets handiger aangezien we dat overal gebruiken (hoewel TOML een fijn format is). Maar heeft geen prio, dus vergeet dit maar even :)